### PR TITLE
Use GitHub Actions Secrets to run FirebaseStorage CI

### PR DESCRIPTION
**Pull request by paulb777**
This PR migrates the Xcode 11 CI for FirebaseStorage to GitHub Actions.

It is the first Firebase iOS usage of GitHub Actions encryption to set up the GoogleServices-Info.plist for the integration tests.

The quickstart tests will follow in a subsequent PR.

Xcode 10 testing will remain on travis.

Xcode 11 testing will run on both until GitHub Actions demonstrates its viability a while longer.

#no-changelog

### CODE REVIEW

File: scripts/third_party/travis/retry.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+#COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+#IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+#CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+travis_retry() {
+  local result=0
+  local count=1
+  while [ $count -le 3 ]; do
+    [ $result -ne 0 ] && {
+      echo -e "\n${ANSI_RED}The command \"$@\" failed. Retrying, $count of 3.${ANSI_RESET}\n" >&2
```
Comment (wilhuff) [inline scripts/third_party/travis/retry.sh]: This is missing definitions for things like ANSI_RED and ANSI_RESET.

I suspect there's more of the library that we're missing.

Comment (paulb777) [reply scripts/third_party/travis/retry.sh]: Fixed - and added the source origin - thanks

**Comment by paulb777** [comment]
Travis failure is addressed by #4730

**Comment by schmidt-sebastian** [comment]
I don't have any context on this (yet). @ryanwilson - is this something you can review? Otherwise, I can spend some time figuring out how this configuration works.

**Comment by paulb777** [comment]
@schmidt-sebastian Should be fine, I primarily added you to reviewers to keep you in the loop. It's still early days for GitHub Actions while iCore explores and work out kinks.

**Pull request merged**